superbit_lensing/shear_profiles/shear_plots.py

```diff
@@ -0,0 +1,163 @@
+            #data.remove_rows([0])
+            radius=data['r']
+            radius=radius*pixscale/60.
+
+        try:
+            etan=data['gtan']
+            ecross=data['gcross']
+            shear1err=data['err_gtan']
+            shear2err=data['err_gcross']
+
+        except KeyError:
+
+            etan=data['col3']
+            ecross=data['col4']
+            shear1err=data['col5']
+            shear2err=data['col6']
+
+        # So far, this is looking great!!! Now, let's remember how to make bin width error bars
+        minrad = minrad*pixscale/60 #pixels --> arcmin
+        maxrad = maxrad*pixscale/60 #pixels --> arcmin
+
+        upper_err=np.zeros_like(radius)
+        for e in range(len(radius)-1):
+            this_err=(radius[e+1]-radius[e])*0.5
+            upper_err[e]=this_err
+        upper_err[-1]=(maxrad-radius[-1])*0.5
+
+        lower_err=np.zeros_like(radius)
+        for e in (np.arange(len(radius)-1)+1):
+            this_err=(radius[e]-radius[e-1])*0.5
+            lower_err[e]=this_err
+
+        lower_err[0]=(radius[0]-minrad)*0.5
+
+        # And scene
+        rad_err=np.vstack([lower_err,upper_err])
+
+        rcParams['axes.linewidth'] = 1.3
+        rcParams['xtick.labelsize'] = 16
+        rcParams['ytick.labelsize'] = 16
+        rcParams['xtick.minor.visible'] = True
+        rcParams['xtick.minor.width'] = 1
+        rcParams['xtick.direction'] = 'inout'
+        rcParams['ytick.minor.visible'] = True
+        rcParams['ytick.minor.width'] = 1
+        rcParams['ytick.direction'] = 'out'
+
+        fig, axs = plt.subplots(2, 1, figsize=sizesharex=True)#,sharey=True)
+        fig.subplots_adjust(hspace=0.1)
+
+        axs[0].errorbar(radius, etan, yerr=shear1err, xerr=rad_err, fmt='-o',
+                        capsize=5, color='cornflowerblue', label=label)
+        axs[0].axhline(y=0,c="black",alpha=0.4,linestyle='--')
+        axs[0].set_ylabel(r'$g_{+}(\theta)$',fontsize=16)
+        axs[0].tick_params(which='major',width=1.3,length=8)
+        axs[0].tick_params(which='minor',width=0.8,length=4)
+        axs[0].set_title(title,fontsize=14)
+        axs[0].set_ylim(-0.05,0.60)
+
+        axs[1].errorbar(radius,ecross,xerr=rad_err,yerr=shear2err,fmt='d',capsize=5,color='cornflowerblue',alpha=0.5,label=label)
+        axs[1].axhline(y=0,c="black",alpha=0.4,linestyle='--')
+        axs[1].set_xlabel(r'$\theta$ (arcmin)',fontsize=16)
+        axs[1].set_ylabel(r'$g_{\times}(\theta)$',fontsize=16)
+        axs[1].tick_params(which='major',width=1.3,length=8)
+        axs[1].tick_params(which='minor',width=0.8,length=4)
+        axs[1].set_ylim(-0.1, 0.1)
+        axs[1].legend()
+
+        if title is None:
+            title = r'S2N\_R$>$5 T$>$ 1.2*TPsf $0.02<T<10$ 1E-2 covcut'
+
+        if outfile is not None:
+            plt.savefig(outfile, bbox_inches='tight', dpi=300)
+
+        if show is True:
+            plt.show()
+
+        return
+
+def covar_calculations(nfw, radius, etan, variance):
```

Comment:
This function is where our figure of merit `alpha` gets calculated, and I'd like your input on how it's written. In particular, I'm worried I've done something wrong in the `np.interp` step, since the value of the shear bias will depend on the number of radial bins. 

Even if it is correct, this linchpin of our analysis seems a bit buried in this `shear_plots.py` code. I'm open to suggestion on where else to do this calculation.

Reply:
I'm actually going to make this a separate issue